test/dialect/test_sqlite.py

```diff
@@ -2297,10 +2299,63 @@ def test_create_index_with_schema(self):
                     "unique": 0,
                     "name": "ix_main_l_bar",
                     "column_names": ["bar"],
+                    "dialect_options": {},
                 }
             ],
         )
 
+    def test_reflect_partial_indexes(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

use the connection fixture

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4200